src/Apps/Order/routes.tsx

```diff
@@ -68,6 +68,9 @@ export const routes: RouteConfig[] = [
             }
           }
         `,
+        cacheConfig: {
+          force: true,
+        },
```

Comment:
Whenever this route is matched it will always re-request data.

Reply:
Ahhh cool. I think we actually want this for _all_ routes in the checkout flow, so I may move it up a level in a subsequent PR.